features/main/table_inheritance.feature

```diff
@@ -80,7 +84,40 @@ Feature: Table inheritance
     }
     """
 
-  @createSchema
+  Scenario: Some children not api resources are created in the app
+    When some dummy table inheritance data but not api resource child are created
+    And I send a "GET" request to "/dummy_table_inheritances"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+    And the JSON should be valid according to this schema:
+    """
+    {
+      "type": "object",
+      "properties": {
+        "hydra:member": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "@type": {
+                "type": "string",
+                "pattern": "^DummyTableInheritance(Child)?$",
```

Comment:
We're not really testing there is at least one child, right?

Reply:
I don't understand why the dependency between scenarios is an issue here.
You're right about the first part though. But maybe we can avoid this for this scenario?

Reply:
> I don't understand why the dependency between scenarios is an issue here.

It's always an issue when you want to run 1 failing test case without running the whole suite. tests should respect the FIRST principle.

Reply:
Sure, I agree with this. But I don't why the link with what I said.

Reply:
Yes sorry you're right. IIUC you're saying we should either have `^DummyTableInheritanceChild$` or `^DummyTableInheritance$` ? 
IIRC there may be both types? If not we definitely need to pick one.

Reply:
@alanpoulain oh, it's possible to be sure the item is here at least once? How?

Reply:
you might be able use tuple validation maybe ? (that's only 3 items... https://json-schema.org/understanding-json-schema/reference/array.html#tuple-validation)

Reply:
We will use multiple `"contains"`: https://json-schema.org/latest/json-schema-validation.html#rfc.section.6.4.6